bitnami/cilium/templates/_helpers.tpl

```diff
@@ -0,0 +1,212 @@
+{{- end -}}
+
+{{/*
+Return the proper Cilium key-value store fullname
+*/}}
+{{- define "cilium.kvstore.fullname" -}}
+{{- include "common.names.dependency.fullname" (dict "chartName" "etcd" "chartValues" .Values.etcd "context" $) -}}
+{{- end -}}
+
+{{/*
+Return the proper Cilium Agent image name
+*/}}
+{{- define "cilium.agent.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.agent.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper Cilium Operator image name
+*/}}
+{{- define "cilium.operator.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.operator.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper Cilium Operator image name
+*/}}
+{{- define "cilium.envoy.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.envoy.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "cilium.imagePullSecrets" -}}
+{{- include "common.images.renderPullSecrets" (dict "images" (list .Values.agent.image .Values.operator.image .Values.envoy.image) "context" $) -}}
+{{- end -}}
+
+{{/*
+Return the Cilium configuration configmap.
+*/}}
+{{- define "cilium.configmapName" -}}
+{{- if .Values.existingConfigmap -}}
+    {{- print (tpl .Values.existingConfigmap $) -}}
+{{- else -}}
+    {{- print (include "common.names.fullname" .) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the Cilium configuration configmap.
+*/}}
+{{- define "cilium.envoy.configmapName" -}}
+{{- if .Values.envoy.existingConfigmap -}}
+    {{- print (tpl .Values.envoy.existingConfigmap $) -}}
+{{- else -}}
+    {{- print (include "cilium.envoy.fullname" .) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for Cilium Agent
+*/}}
+{{- define "cilium.agent.serviceAccountName" -}}
+{{- if .Values.agent.serviceAccount.create -}}
+    {{ default (include "cilium.agent.fullname" .) .Values.agent.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.agent.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for Cilium Operator
+*/}}
+{{- define "cilium.operator.serviceAccountName" -}}
+{{- if .Values.operator.serviceAccount.create -}}
+    {{ default (include "cilium.operator.fullname" .) .Values.operator.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.operator.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for Cilium Envoy
+*/}}
+{{- define "cilium.envoy.serviceAccountName" -}}
+{{- if .Values.envoy.serviceAccount.create -}}
+    {{ default (include "cilium.envoy.fullname" .) .Values.envoy.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.envoy.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the name of the secret containing the TLS CA
+*/}}
+{{- define "cilium.tls.caSecretName" -}}
```

Comment:
This helper is not used anywhere

Reply:
addressed at https://github.com/bitnami/charts/pull/26374/commits/d6b150377935aae125234d5476f88708f3d93c5e